backend/src/crd/controller/scheduledworkflow/main.go

```diff
@@ -106,6 +110,8 @@ func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&namespace, "namespace", "", "The namespace name used for Kubernetes informers to obtain the listers.")
+	flag.Float64Var(&clientQPS, "clientQPS", 5, "The maximum QPS to the master from this client.")
```

Comment:
Out of interest is there a reason for using `clientQPS` here and `client_qps` here: https://github.com/kubeflow/pipelines/blob/4373874925290230740508efaa47f1944528cc4c/backend/src/crd/controller/viewer/main.go#L49

Reply:
Just try to keep consistent inside the component.

Reply:
Ok, thanks for the clarification.